tests/security/ima/evm_protection_digital_signatures.pm

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test EVM protection using digital signatures
+# Note: This case should come after 'evm_protection_hmacs'
+# Maintainer: wnereiz <wnereiz@member.fsf.org>
+# Tags: poo#53582
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use bootloader_setup "replace_grub_cmdline_settings";
+use power_action_utils "power_action";
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $fstype     = 'ext4';
+    my $sample_app = '/usr/bin/yes';
+    my $sample_cmd = 'yes --version';
+
+    my $mok_priv = '/root/certs/key.asc';
+    my $mok_pass = "suse";
```

Comment:
It could be graceful to be consistent with `"` or `'`  when defining variables.

Reply:
Done, thanks!